### ISSUE TITLE
Results Analysis Scripts

### ISSUE THREAD

**Issue by eddableheath** [alan-turing-institute/ARC-LoCoMoSeT#60]
We have some scripts for loading/parsing the results data and making plots with already, however not only do these require updating as our results format has changed, we also need some further analysis:

- Metric evaluation metric comparisons.
- Aggregation of metric evaluation scores across datasets/models.
- Combinations of metrics? #42 
- Weighting metrics by task agnostic metrics? #37 
- Further plots of cross-sectional analysis of our experiments.
- Time analysis #86 
- Variation with dataset size (for training and metrics)
- ImageNet validation accuracy as a metric #104

It would be useful to have a script(s) ready that can run these once we have the validation accuracy scores from the fine-tuning experiments.

**Comment by eddableheath** [comment]
I've put this in work package 3B for now, but scope to move this forward to 3A or back to 4.

**Comment by eddableheath** [comment]
This issue is also linked with - and should follow - #62

**Comment by eddableheath** [comment]
I've assigned myself to this for now as I've got a lot of snippets of things in a notebook that need to be pulled out into full scripts. I'll start by putting that notebook an a branch for this issue.
